src/index.html

```diff
@@ -16,8 +16,8 @@
   </head>
   <body>
     <header class="header">
```

Comment:
Great job @edi-sipka 🤝

- [ ] [Optional] You may kindly style your UI so that it resembles the design provided. However, this is not required at this stage but it will look your design more appealing.